api/features/import_export/tasks.py

```diff
@@ -200,3 +204,33 @@ def _create_new_feature(
     )
     feature_state.enabled = feature_data["enabled"]
     feature_state.save()
+
+
+@register_recurring_task(
+    run_every=timedelta(hours=24),
+)
+def create_flagsmith_on_flagsmith_feature_export():
+    if (
+        not settings.FLAGSMITH_ON_FLAGSMITH_FEATURE_EXPORT_ENVIRONMENT_ID
+        or not settings.FLAGSMITH_ON_FLAGSMITH_FEATURE_EXPORT_TAG_ID
+    ):
+        # No explicit settings on both feature settings, hence exit.
```

Comment:
I'd suggest that probably the settings filter should be before we register the recurring task, otherwise it might be a little odd for self hosted people that see the task registered. 

e.g. 

```python
if (settings.FLAGSMITH_ON_FLAGSMITH_FEATURE_EXPORT_ENVIRONMENT_ID and settings.FLAGSMITH_ON_FLAGSMITH_FEATURE_EXPORT_TAG_ID):
    @register_recurring_task(
        run_every=timedelta(hours=24),
    )
    def create_flagsmith_on_flagsmith_feature_export():
        ...
```

Reply:
As we chatted briefly on Slack, there is the issue of importing this so in the mean time I've wrapped a private function and used it in conjunction with your suggestion to keep the task from queuing for the other clients.